backend/hitas/calculations/improvements/rules_pre_2011_mpi.py

```diff
@@ -219,6 +219,19 @@ class HousingCompanyImprovementCalculationResult(ApartmentImprovementCalculation
 
 @dataclass
 class HousingCompanyImprovementCalculationSummary(ApartmentImprovementCalculationSummary):
+    @dataclass
+    class ExcessSummary:
+        # Apartment's surface area
+        surface_area: Decimal
+        # Excess amount per square meter before 2010/01/01
+        value_per_square_meter_before_2010: Decimal
+        # Excess amount per square meter on or after 2010/01/01
+        value_per_square_meter_after_2010: Decimal
```

Comment:
I would say no need to save these, as they are hardcoded values

Reply:
I think we end up saving more if we save it for every improvement, and it's also easier to write the notes if the values are included in the excess data. Improvement already get's the `completion_date` so I can use that and add an `improvements_before_2010: bool` and `improvements_after_2010: bool` to the excess. Or I'll make the `value_per_square_meter_before_2010` and `value_per_square_meter_after_2010` nullable and determine based on that.

Reply:
Made corresponding `value_per_square_meter` and `total` fields null if said excess is not applicable. This allows them to be hidden as well in the PDF.